src/Pmi/Service/EmailNotificationService.php

```diff
@@ -1,135 +0,0 @@
-<?php
```

Comment:
This appeared to no longer be in use, as everything related to it has been moved over to the Symfony namespace.